packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -458,6 +481,21 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
               }, 0)`}
               className={this._classNames.yAxis}
             />
+            <SVGTooltipText
+              content={this.props.yAxisTitle}
+              textProps={{
+                x: this.margins.bottom! + this.state._removalValueForTextTuncate! + this.titleMargin,
+                y: this._isRtl
+                  ? svgDimensions.width - this.margins.right! / 2 + this.titleMargin
+                  : this.margins.left! / 2 + this.state.startFromX - this.titleMargin,
+                textAnchor: 'middle',
+                transform: `translate(0,
+                   ${svgDimensions.height - this.margins.bottom! - this.margins.top! - this.titleMargin})rotate(-90)`,
+                className: this._classNames.tooltipTitle!,
+              }}
+              maxWidth={yAxisTitleMaximumAllowedHeight}
+              wrapContent={wrapContent}
+            />
             {this.props.secondaryYScaleOptions && (
```

Comment:
Add title for secondary y axis also

Reply:
done:
<img width="500" alt="image" src="https://github.com/microsoft/fluentui/assets/120183316/716b9523-4c86-4a42-9269-2893343f182d">